plugins/scaffolder-backend/package.json

```diff
@@ -79,7 +79,8 @@
     "winston": "^3.2.1",
     "yaml": "^2.0.0",
     "zen-observable": "^0.10.0",
-    "zod": "~3.18.0"
+    "zod": "~3.18.0",
+    "zod-to-json-schema": "~3.18.0"
```

Comment:
We normally use `"zod-to-json-schema": "^3.18.0"`

Reply:
I tried using `"zod-to-json-schema": " ^3.18.0"` but it fails in the pipeline throwing a tsc error `Generic type 'ZodDiscriminatedUnionDef<Discriminator, DiscriminatorValue, Option>' requires 3 type argument(s)` when running `yarn tsc:full`